src/styles/components.less

```diff
@@ -29,4 +29,4 @@
 @import '../components/Tabs/Tabs';
 @import '../components/Overlay/Overlay';
 @import '../components/Dialog/Dialog';
-
+@import '../components/Autocomplete/Autocomplete'; // Should import after DropMenu
```

Comment:
We should hoist the variables into `variable.less` rather than continuing to do this. Filed #148 for tech debt.